setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-license_file = LICENSE.txt
+license_files = LICENSE.txt
```

Comment:
Why?

Reply:
It is explained in the commit message which is not immediately seen in GitHub, I can split it into a separate PR, just like the other unrelated patches.

https://github.com/hardbyte/python-can/pull/1396/commits/8e9b8fc49a0f2706b2553981ee08ec9c25258244

Reply:
It is good to keep PR’s within the narrowest scope as practical. Perhaps a quote by Einstein somewhat applies; “Everything should be made as simple as possible, but not simpler.” The review process is simpler (and often can occur quicker) if changes fall within a scope described in the related issue.

Reply:
OK, I removed the other patches and will post them separately.